### PR TITLE
chore: apply post release modifications v0.6.0

### PR DESCRIPTION
**Pull request by felixdittrich92**
This PR:

- add post release modifications
- set dev to 0.7.0

### CODE REVIEW

**Review by charlesmindee — APPROVED** [review]
thanks!

**Review by odulcy-mindee — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from setuptools import setup
 
 PKG_NAME = "python-doctr"
-VERSION = os.getenv("BUILD_VERSION", "0.6.0a0")
+VERSION = os.getenv("BUILD_VERSION", "0.7.0a0")
```
Comment (frgfm) [inline setup.py:12]: 0.6.1a0 out of safety ;)

**Review by frgfm — COMMENTED** [review]
I think we shouldn't stress ourselves by setting a version index too far. If we release X.Y.0, the dev version should be something like X.Y.1a0 (otherwise if we release a patch, that will mess up some dev installations for a few people)

File: setup.py (hunk 1 above)
Comment (felixdittrich92) [reply setup.py:12]: yeah i set it to 0.7.0 because we have no backlog. But you are right :)

**Pull request merged**